.github/workflows/milestone-merged-prs.yaml

```diff
@@ -0,0 +1,15 @@
+name: Milestone
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+jobs:
+  milestone_pr:
+    name: attach to PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: scientific-python/attach-next-milestone@main
```

Comment:
We probably want to pin this once the action is more stable and mature...

Reply:
Indeed!